pkg/app/master/command/images/handler.go

```diff
@@ -131,17 +133,17 @@ func OnCommand(
 	if cparams.TUI { // `images --tui`
 		initialTUI := InitialTUI(images, true)
 		tui.RunTUI(initialTUI, true)
-		return nil
 	} else if cparams.GlobalTUI { // `tui` -> `i`
-		return images
+		// TODO - create a central store for the lookup key.
+		// As this key needs to be the same on the sender and the receiver.
+		xc.Out.DataChannels["images"] <- images
+		close(xc.Out.DataChannels["images"])
```

Comment:
We should revisit who is responsible for closing the channels especially if the channels are not exposed here. TBD...

Reply:
Yeah will certainly not be closed here as per your above comment about removing `handler.go`'s awareness of the channel